src/controller/users.js

```diff
@@ -15,7 +14,7 @@ const loginUser = async (req, res) => {
     const auth = User.authenticate();
     const { user, error } = await auth(username, password);
     if (user) {
-      const accessToken = jwt.sign({ name: user.username, exp: expirationDate }, secret);
+      const accessToken = jwt.sign(user.toJSON(), secret);
```

Comment:
Dont we want to set expiry date for the tokens? It is really dangerous. I would go with an increased expiry date instead